costs/tags/tags_untagged.go

```diff
@@ -0,0 +1,170 @@
+//   Copyright 2017-2018 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package tags
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/trackit/jsonlog"
+	"github.com/trackit/trackit-server/errors"
+	"github.com/trackit/trackit-server/es"
+	"gopkg.in/olivere/elastic.v5"
```

Comment:
Please move `"gopkg.in/olivere/elastic.v5"` below  `"github.com/trackit/jsonlog"`

Reply:
This import is deprecated, please use `"github.com/olivere/elastic"`